### ISSUE TITLE
Value upacking error in getGatewayIP()

### ISSUE THREAD

**Issue by rhodesn** [k4m4/kickthemout#438]
## Checklist for submitting an issue to `KickThemOut`:

- [x] I have carefully read the [README](https://github.com/k4m4/kickthemout/blob/master/README.rst) file and haven't managed to resolve my issue.
- [x] I have searched the [issues](https://github.com/k4m4/kickthemout/issues?utf8=%E2%9C%93&q=is%3Aissue) of this repo and believe that this is not a duplicate.
- [x] I am running the latest version of KickThemOut.

 

<br/ >

- **Description**: Unable to retrieve gateway IP

I'm seeing an unpacking error in `getGatewayIP()`:
```python
>>> from scapy import *
>>> from scapy.all import *
>>> sr1(IP(dst="github.com", ttl=0) / ICMP() / "XXXXXXXXXXX", verbose=False, timeout=4)
<IP  version=4 ihl=5 tos=0xc0 len=67 id=45049 flags= frag=0 ttl=64 proto=icmp chksum=0x45e6 src=192.168.1.1 dst=192.168.1.201 |<ICMP  type=time-exceeded code=ttl-zero-during-transit chksum=0xf4ff reserved=0 length=0 unused=None |<IPerror  version=4 ihl=5 tos=0x0 len=39 id=1 flags= frag=0 ttl=0 proto=icmp chksum=0xf60d src=192.168.1.201 dst=140.82.118.4 |<ICMPerror  type=echo-request code=0 chksum=0xe645 id=0x0 seq=0x0 |<Raw  load='XXXXXXXXXXX' |>>>>>
>>> getGateway, timeout = sr1(IP(dst="github.com", ttl=0) / ICMP() / "XXXXXXXXXXX", verbose=False, timeout=4)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: not enough values to unpack (expected 2, got 1)
>>>
```

I've had a quick play, and was wondering if there is a reason you are not using `conf.route.route('0.0.0.0')[2]` to return the default gateway?

```diff
diff --git a/kickthemout.py b/kickthemout.py
index afeafd4..cdd50a2 100644
--- a/kickthemout.py
+++ b/kickthemout.py
@@ -185,10 +185,7 @@ def getDefaultInterfaceMAC():
 def getGatewayIP():
     global stopAnimation
     try:
-        getGateway, timeout = sr1(IP(dst="github.com", ttl=0) / ICMP() / "XXXXXXXXXXX", verbose=False, timeout=4)
-        if timeout:
-            raise Exception()
-        return getGateway.src
+        return conf.route.route('0.0.0.0')[2]
     except:
         # request gateway IP address (after failed detection by scapy)
         stopAnimation = True
```

Basically the result of the unpacking error is that I am always prompted to provide my gateway IP.

**Comment by k4m4** [comment]
Thanks for bringing this up @nrhodes91. I'll need to have another look at the scapy docs.
